lib/pages/settings/view/request_card.dart

```diff
@@ -51,7 +54,8 @@ class _RequestCardState extends State<RequestCard>
                             .copyWith(fontSize: 14),
                       ),
                       const SizedBox(height: 10),
-                      _buildButtons(request.processed, adminProvider, request)
+                      _buildButtons(request.processed, adminProvider,
```

Comment:
You don't need to pass the providers, you can just do Provider.of(...) directly in the helper method.